util/cron/test-perf.cray-cs.ofi.bash

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+#
+# Run performance tests with comm=ofi on a cray-cs system.
+
+CWD=$(cd $(dirname $0) ; pwd)
+
+export CHPL_TEST_PERF_CONFIG_NAME='16-node-cs'
+
+source $CWD/common-perf.bash
+
+export CHPL_NIGHTLY_TEST_CONFIG_NAME="perf.cray-cs.ofi"
+
+module load gcc
+module load python/2.7.6
+module load pmix
+
+export CHPL_HOST_PLATFORM=cray-cs
+export CHPL_COMM=ofi
+export CHPL_COMM_OFI_OOB=slurm-pmi2
+export SLURM_MPI_TYPE=pmix
+export CHPL_RT_MAX_HEAP_SIZE=83G
+export CHPL_LAUNCHER=slurm-srun
+export CHPL_LAUNCHER_PARTITION=bdw18
+export CHPL_TARGET_CPU=broadwell
+
+source /cray/css/users/chapelu/setup_libfabric.bash || return 1
+
+nightly_args="${nightly_args} -no-buildcheck"
+
+perf_args="-performance-description ofi -performance-configs gn-ibv-large,gn-ibv-fast:v,gn-mpi,ofi:v"
```

Comment:
`-performance-configs` should match across all the configs (and cron files) that we want to graph together. If we want to graph ofi and all 3 gasnet configs side-by-side this should be: `-performance-configs gn-ibv-large:v,gn-ibv-fast:v,gn-mpi,ofi:v` (where `:v` means the config should be visible by default.  If you look at https://chapel-lang.org/perf/16-node-cs/?suite=performancetracking, you'll see that by default only ibv-large and ibv-fast are visible, but you can toggle mpi.)

I'm not sure if we want the ofi results to be in with the normal 16-node-cs graphs or if we want them to be in a separate directory. If we want results all 4 configs to be at https://chapel-lang.org/perf/16-node-cs, then just copy that `-performance-configs to ofi and the 3 gasnet configs. If you want them to be at something like https://chapel-lang.org/perf/16-node-cs/ofi instead I think we can add `-sync-dir-suffix ofi`

Reply:
In a follow-up PR I am going to set things up so that for now, comm=ofi will be graphed with the 3 existing comm=gasnet configs.